frontend/src/Pages/Fits/FittingSortDisplay.js

```diff
@@ -257,7 +257,7 @@ function ImplantOut() {
     <>
       <DisplayDOM style={{ justifyContent: "initial" }}>
         <ImplantButton name="Ascendancy" img={wbadge} />
-        <ImplantButton name="Hybrid" img={hbadge} />
+        <ImplantButton name="Amulet" img={hbadge} />
```

Comment:
Probably want to get rid of the H and replace with an A for amulate

Reply:
Points to the hardwire image, so it's the same no matter which one it is.